lib/safemode/core_jails.rb

```diff
@@ -94,56 +94,114 @@ def core_jail_class_methods(klass)
   # whitelisted methods for core classes ... kind of arbitrary selection
   @@methods_whitelist = {
   'Array' => %w(
+    all?
     any?
+    append
     assoc
     at
-    blank?
```

Comment:
Some methods were moved to the `default_methods` so they are deleted from individual class lists.